### PR TITLE
feature: Convert tests to doctests and add example

### PR DESCRIPTION
**Pull request by Byambaa0325**
Hello, 
I am probably little late here but I converted some tests in test_features.py to doctest examples in feature.py.
I wanted to confirm whether it is the desired result of the Doctest-able Examples GSOC project?

### CODE REVIEW

**Comment by Byambaa0325** [comment]
@pdxjohnny (tagging for a comment)
Mentioned project: [Doctest-able Examples](https://github.com/intel/dffml/issues/366)

File: dffml/feature/feature.py [hunk 1]
```diff
@@ -155,6 +183,13 @@ def length(self) -> int:
         """
         Models need to know a Feature's length, 1 means single value, more than
         that is the length of the array calc returns.
+        
+        Default example
```
Comment (pdxjohnny) [inline dffml/feature/feature.py]:
```suggestion
       Examples
```

Comment (pdxjohnny) [reply dffml/feature/feature.py]: For consistency we always use

```
        Examples
        --------
```

Comment (Byambaa0325) [reply dffml/feature/feature.py]: Got it!

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Ability to run a subflow as if it were an operation using the
   `dffml.dataflow.run` operation.
 - Support for operations without inputs.
+- Partial doctestable examples to `features.py`
+
```
Comment (pdxjohnny) [inline CHANGELOG.md:21]:
```suggestion
```

We want to maintain consistency with the rest of the sections by not adding a blank line here

**Comment by pdxjohnny** [comment]
@Byambaa0325 Yes you're on the right track here!

**Pull request merged**